internal/api/deployments.go

```diff
@@ -24,7 +24,7 @@ type Deployment struct {
 	ConcurrencyLimit       *int64                         `json:"concurrency_limit"`
 	ConcurrencyOptions     *ConcurrencyOptions            `json:"concurrency_options"`
 	Description            string                         `json:"description"`
-	EnforceParameterSchema bool                           `json:"enforce_parameter_schema"`
+	EnforceParameterSchema *bool                          `json:"enforce_parameter_schema"`
```

Comment:
This addresses a problem between Cloud and OSS where the default value here is different. We need to use a pointer to differentiate between unset and intentionally setting `false`.